core_backend/migrations/versions/e8ddc3de6210_create_content_feedback_tables.py

```diff
@@ -0,0 +1,108 @@
+"""Create Content Feedback Tables
+
+Revision ID: e8ddc3de6210
+Revises: 50caccd9e332
+Create Date: 2024-04-19 15:36:53.671146
+
+"""
+
+from typing import Sequence, Union
+
+import sqlalchemy as sa
+from alembic import op
+from sqlalchemy.dialects import postgresql
+
+# revision identifiers, used by Alembic.
+revision: str = "e8ddc3de6210"
+down_revision: Union[str, None] = "50caccd9e332"
+branch_labels: Union[str, Sequence[str], None] = None
+depends_on: Union[str, Sequence[str], None] = None
+
+
+def upgrade() -> None:
+    # ### commands auto generated by Alembic - please adjust! ###
+    op.create_table(
+        "content_feedback",
+        sa.Column("feedback_id", sa.Integer(), nullable=False),
+        sa.Column("feedback_sentiment", sa.String(), nullable=True),
+        sa.Column("query_id", sa.Integer(), nullable=False),
+        sa.Column("feedback_text", sa.String(), nullable=True),
+        sa.Column("feedback_datetime_utc", sa.DateTime(), nullable=False),
+        sa.Column("content_id", sa.Integer(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["content_id"],
+            ["content.content_id"],
+        ),
+        sa.ForeignKeyConstraint(
+            ["query_id"],
+            ["user-queries.query_id"],
+        ),
+        sa.PrimaryKeyConstraint("feedback_id"),
+    )
+    op.create_index(
+        op.f("ix_content_feedback_feedback_id"),
+        "content_feedback",
+        ["feedback_id"],
+        unique=False,
+    )
+    op.create_table(
+        "response_feedback",
+        sa.Column("feedback_id", sa.Integer(), nullable=False),
+        sa.Column("feedback_sentiment", sa.String(), nullable=True),
+        sa.Column("query_id", sa.Integer(), nullable=False),
+        sa.Column("feedback_text", sa.String(), nullable=True),
+        sa.Column("feedback_datetime_utc", sa.DateTime(), nullable=False),
+        sa.ForeignKeyConstraint(
+            ["query_id"],
+            ["user-queries.query_id"],
+        ),
+        sa.PrimaryKeyConstraint("feedback_id"),
+    )
+    op.create_index(
+        op.f("ix_response_feedback_feedback_id"),
+        "response_feedback",
+        ["feedback_id"],
+        unique=False,
+    )
+    op.drop_index("ix_feedback_feedback_id", table_name="feedback")
+    op.drop_table("feedback")
```

Comment:
We are recreating the tables! 
This is what alembic does by default. I think this is not an issue since this is not deployed and we don't have feedback we are trying to preserve. If it is an issue, we'll have to modify this to do a `rename` instead - a lot messier.